class.jetpack-cli.php

```diff
@@ -118,6 +118,10 @@ public function test_connection( $args, $assoc_args ) {
 		}
 
 		$body = wp_remote_retrieve_body( $response );
+		if ( is_wp_error( $body ) ) {
```

Comment:
This is not ok because `wp_remote_retrieve_body` never returns an instance of `WP_Error`.
Instead, `is_wp_error` is used inside its code to decide whether to return an empty string or the body, which again, it's a string
Here's the documentation for `wp_remote_retrieve_body` which also include its source code where what I mentioned here can be seen
https://developer.wordpress.org/reference/functions/wp_remote_retrieve_body/